ecl/agentexec/agentexec.cpp

```diff
@@ -20,81 +20,67 @@
 #include "jmisc.hpp"
 #include "jlog.hpp"
 #include "jfile.hpp"
-#include "agentexec.hpp"
 #include "jutil.hpp"
 #include "eclagent.hpp"
 
-Owned<CEclAgentExecutionServer> execSvr = NULL;
-
-
-//---------------------------------------------------------------------------------
-
-CEclAgentExecutionServer::CEclAgentExecutionServer(IPropertyTree *_config) : Thread("Workunit Execution Server"), config(_config)
-{
-    started = false;
-}
-
-CEclAgentExecutionServer::~CEclAgentExecutionServer()
+class CEclAgentExecutionServer
+#ifdef _CONTAINERIZED
```

Comment:
minor: I think I would be tempted for this to be unconditional, and have the contents of createNew() inside a #if

Reply:
I will change, though not sure it's any clearer.